bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,298 @@ class CoffeeLintBear:
+        :param: use_english_operator:
+            Determines if ``and``, ``or``, ``is`` and ``isnt`` should be used
+            instead of ``&&``, ``||``, ``==`` and ``!=``.
+        :param: spaces_around_operators:
+            Enforces that operators have spaces around them.
+        :param: space_after_comma:
+            Checks if there is a blank space after commas.
+        :param: cyclomatic_complexity:
+            Determines if the complexity of the file should be examined.
+        :param: cyclomatic_complexity_value:
+            Cyclomatic complexity of the file.
+        :param: prevent_duplicate_keys:
+            Prevents defining duplicate keys in object literals and classes.
+        :param: enforce_parentheses_on_non_empty_constructors:
+            Requires constructors with parameters to include parentheses.
+        :param: consistent_line_endings:
+            Ensures that the file uses only ``windows`` or ``unix`` line
+            endings.
+        :param: consistent_line_endings_style:
+            The option to ``line_endings``, its value is either ``unix`` or
+            ``windows``.
+        :param: no_this:
+            Prohibits ``this``. Uses ``@`` instead.
+        :param: no_decr_or_incrementation_operators:
+            Prohibits the use of increment and decrement arithmetic operators.
+        :param: no_empty_parameter_list:
+            Prohibits empty parameter lists in function definitions.
+        :param: no_empty_functions:
+            Prohibits declaring empty functions.
+        """
+        coffee_configs = {"max_line_length":
+                          {"value": max_line_length,
+                           "level": "error",
+                           "limitComments":
+                               max_line_length_affects_comments}}
+        if space_before_and_after_arrow:
+            coffee_configs["arrow_spacing"] = {"level": "error"}
+        if check_braces_spacing:
+            coffee_configs["braces_spacing"] = (
+                {"level": "error",
+                 "spaces": braces_spacing_width,
+                 "empty_object_spaces": spacing_in_empty_braces})
+        if camelcase_classes_name:
+            coffee_configs["camel_case_classes"] = {"level": "error"}
+        if spaces_before_and_after_colon:
+            coffee_configs["colon_assignment_spacing"] = (
+                {"level": "error",
+                 "spacing": {"left": spaces_before_colon,
+                             "right": spaces_after_colon}})
+        if check_whether_file_ends_in_newline:
+            coffee_configs["eol_last"] = {"level": "error"}
+        if newlines_after_classes:
+            coffee_configs["newlines_after_classes"] = (
+                {"value": number_of_newlines_after_classes,
+                 "level": "error"})
+        if no_backticks:
+            coffee_configs["no_backticks"] = {"level": "error"}
+        else:
+            coffee_configs["no_backticks"] = {"level": "ignore"}
```

Comment:
this can be shortened to `coffee_configs['no_backticks'] = {'level': 'ignore' if no_backticks else 'error'}`